cmd/kubeadm/app/phases/addons/dns/dns_test.go

```diff
@@ -632,7 +632,9 @@ func TestCreateCoreDNSConfigMap(t *testing.T) {
     }`,
 			expectedCorefileData: `.:53 {
     errors
-    health
+    health {
+        lameduck 5s
```

Comment:
is this a required change? will users with a custom dns config be broken if they don't make this change as well?

Reply:
It's not required.  It's just an improvement that reduces query failures during rolling upgrades.  The setting allows CoreDNS to complete in flight dns queries before exiting.

Without the setting, CoreDNS will not be broken.